lib/transforms/buildProduction.js

```diff
@@ -284,12 +284,15 @@ module.exports = function (options) {
             .endif()
             .if(!options.noFileRev)
                 .moveAssetsInOrder(query.and(
-                    {
-                        isLoaded: true,
-                        isInline: false,
-                        type: query.not(['CacheManifest', 'Rss', 'Atom']),
-                        fileName: query.not(['.htaccess', 'humans.txt', 'robots.txt'])
-                    },
+                    query.or(
+                        {type: 'Html', isFragment: true, isLoaded: true},
```

Comment:
`isFragment` is not the same as `isInline`. Fragments are HTML assets without an `<html>` element.